api/python/provisioner/inputs.py

```diff
@@ -877,3 +879,71 @@ def is_iso(self):
 @attr.s(auto_attribs=True)
 class SWUpgradeRepo(SWUpdateRepo):
     _param_di = param_spec['swupgrade/repo']
+    target_build: str = attr.ib(
```

Comment:
who is expected to provide that value? the meaning of it is actually the base directory for upgrade bundle so we can evaluate all repos locations. So it is expected to be an internal logic:
- if source is ISO then it would be a mount point on the target systems
- if source is DIR then it would be a location the directory  on target systems
- if source is an URL then it would the base dir would match the URL

Reply:
It is internal logic to setup the `target_build` variable

Reply:
looks good now, thanks